CondFormats/CTPPSReadoutObjects/src/classes_def.xml

```diff
@@ -20,4 +20,16 @@
     <class name="CTPPSPixelGainCalibration::DetRegistry"/>
     <class name="std::vector<CTPPSPixelGainCalibration::DetRegistry>"/>
     <class name="CTPPSPixelGainCalibrations" class_version="0"/>
+
+    <class name="PPSTimingCalibration::Key"/>
+    <class name="std::map<PPSTimingCalibration::Key,std::vector<double> >"/>
+    <class name="std::map<PPSTimingCalibration::Key,std::pair<double,double> >"/>
+    <class name="std::pair<PPSTimingCalibration::Key,std::vector<double> >"/>
+    <class name="std::pair<PPSTimingCalibration::Key,std::pair<double,double> >"/>
+    <class name="PPSTimingCalibration" class_version="0">
+      <field name="formula_"/>
+      <field name="parameters_" mapping="blob"/>
```

Comment:
I just realised one of my commits disappeared in the rebase for this PR. Would one expert please comment on the necessity of this `mapping="blob"` flag? The two produced `sqlite` files seem strictly equivalent with and without this blob representation, but this might reflect in the overall performance? (still to be assessed)

Reply:
@forthommel 
mapping="blob" is not required ( but it is not harming )